src/services/users.service.ts

```diff
@@ -52,6 +52,17 @@ class UserService {
     const deleteUserData = await this.users.delete({ where: { id: userId }, select: { email: true, id: true } });
     return deleteUserData;
   }
+
+  public async manageGroups(userId: number, data: any) {
```

Comment:
The name of the function is kinda vague and I think should be `manageUsers`. Also is good to create a DataDTO and put as a data type and add a return type for this function like `Promise<Group>`.